web/src/utils/columns.ts

```diff
@@ -359,8 +375,10 @@ export const getSrcColumns = (t: TFunction): Column[] => {
     },
     {
       id: ColumnsId.srcnamespace,
-      group: t('Source'),
+      group,
       name: t('Namespace'),
+      tooltip: t('The {{group}} project of the related kubernetes resource.', { group: group.toLowerCase() }),
```

Comment:
I'd stick with "Namespace" rather than "Project". It's more agnostic to the kube flavour, project is really an openshift thing